java/.platform.app.yaml

```diff
@@ -4,13 +4,16 @@ type: "java:8"
 disk: 1024
 
 hooks:
-  build: mvn clean install
+  build: mvn clean install -U
```

Comment:
Just for my own education, what's the -U do?

Reply:
The -U seeks to updates/documentation code at the libraries that are a snapshot, on our case, the Java Config-reader.

Reply:
Hm.  Would that bypass a lock file, meaning you may get a new version of some library without knowing it?

Reply:
It will happen just with SNAPSHOT version,
https://github.com/platformsh/language-examples/blob/master/java/pom.xml#L37L42
And the SNAPSHOT to this specific version, 0.0.1, once everything else has a final version there is no way to receive an automatic update.